language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/ImportsAcceptor.java

```diff
@@ -58,28 +60,41 @@ public ImportsAcceptor(DocumentServiceContext context, BiConsumer<String, String
      *
      * @return Returns imports acceptor
      */
-    public BiConsumer<String, String> getAcceptor(DocumentServiceContext context) {
+    public BiConsumer<String, CodeActionModuleId> getAcceptor(DocumentServiceContext context) {
         Optional<Project> project = context.workspace().project(context.filePath());
         String currentPkgName = project.isEmpty() ? "" :
                 CommonUtil.escapeReservedKeyword(project.get().currentPackage().packageName().value());
-        return (orgName, alias) -> {
+        String currentOrgName = project.isEmpty() ? "" :
```

Comment:
Doesn't it means that having an empty project is an error scenario? Also, check whether it is guaranteed that the project exists always. Maybe we need to optimise these APIs in the future to return a project instance instead of an optional and in an error scenario, throwing an exception, because we cannot proceed when the project/ module/ semantic model is not present.

Reply:
Done